.travis.yml

```diff
@@ -17,7 +17,8 @@ before_deploy:
 - chmod +x ./.travis/setup && ./.travis/setup
 ## Add the travis build number to the game engine version
 - GAME_CONFIG=game_engine.properties
-- ENGINE_VERSION=$(grep "engine_version" $GAME_CONFIG | sed 's/.*= *//g')
+- VERSION_SOURCE=src/main/java/games/strategy/engine/config/GameEnginePropertyFileReader.java
+- ENGINE_VERSION=$(grep "public static final String GAME_ENGINE_VERSION *=" $VERSION_SOURCE | sed 's/.*="//g' | cut -d\" -f2)
```

Comment:
Can we blow up the build if `ENGINE_VERSION` is not in the expected format?  Or maybe just verify it's not empty?

I'm only suggesting this because if someone refactors `GameEnginePropertyFileReader`, they may not realize `GAME_ENGINE_VERSION` is being used in a Travis script.  And I can't tell from the rest of the script if an empty `ENGINE_VERSION` will be detected (even by a human) relatively quickly after it is introduced.